common/build/eslint-plugin-fluid/src/rules/no-unchecked-record-access.js

```diff
@@ -39,18 +39,28 @@ module.exports = {
 
 		// Main function to run on every member access (e.g., obj.a or obj["a"])
 		function checkPropertyAccess(node) {
-			// Only check index signature types
+			const fullName = getFullName(node);
```

Comment:
Is there a reason you moved this line to the start?

Reply:
No real reason, I was using it for logging, and it was nice to log at the top with its name